### PR TITLE
add Darshan

### PR DESCRIPTION
**Pull request by darshanime**
Add social info to blog

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/appknox/appknox.github.io/pull/8" target="_blank">ReviewNinja

**Comment by dhilipsiva** [comment]
@jitk14 @subho007 ☝️

File: _data/members.yml [hunk 1]
```diff
@@ -168,3 +168,13 @@
   email: sugat@appknox.com
   github:
   www:
+
+- name: Darshan Chaudhary
+  title: Backend Engineer
+  gravatar: http://www.gravatar.com/avatar/8e4cf01db0775e989082beb30c5cc44c?s=200
```
Comment (ChillarAnand) [inline _data/members.yml]: Can you use `https` instead of `http`?

Comment (dhilipsiva) [inline _data/members.yml]: Thanks for the Catch @ChillarAnand .

**Comment by dhilipsiva** [comment]
@darshanime Could you please make fix the changes suggested by @ChillarAnand ?

**Comment by dhilipsiva** [comment]
@ChillarAnand Please add your details and send a PR. Also, it would be great if you can add your details here as well: https://github.com/appknox/careers#knoxstars

File: _data/members.yml (hunk 1 above)
Comment (subho007) [inline _data/members.yml]: @darshanime waiting for your change

**Comment by darshanime** [comment]
Thanks for the comments. Changed.

**Pull request merged**